system/audit-logs/vendor/fluent-audit-container/templates/prometheus-alerts.yaml

```diff
@@ -12,11 +13,17 @@ metadata:
     app: fluent-audit-container-prometheus
     tier: infra
     type: alerting-rules
-    prometheus: {{ required ".Values.alerts.prometheus missing" $values.alerts.prometheus | quote }}
+    {{- if contains "thanos" $target }}
+    # alerts, which combine metrics from infra-collector and kubernetes, are available from the thanos metal query.
+    {{ $target }}: {{ index $values.alerts.prometheus $target | required (printf "$values.alerts.prometheus.%s missing" $target) }}
```

Comment:
Can this be missing? The loop is only entered if there is any entry in `$values.alerts.prometheus`

Reply:
This is just one solution that you can find on several charts in this repo. But yes - this is not failsafe, so we need to make sure it's there.